<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="VGtab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="VG">
     <attribute name="title">
      <string>VG</string>
     </attribute>
     <widget class="QGroupBox" name="TypeBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sylfaen</family>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>TYPE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="typeText">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="type2Text">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="weakGroup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sylfaen</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>WEAKNESS</string>
      </property>
      <widget class="QLabel" name="normalLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/normal.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="fightLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/fight.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="steelLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/steel.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="waterLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/water.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="fairyLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/fairy.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="groundLabel">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/ground.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="poisonLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/poison.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="rockLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/rock.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="grassLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/Grass.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="ghostLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/ghost.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="flyingLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/flying.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="psychicLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/psychic.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="electricLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/electric.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="bugLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/bug.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="fireLabel">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>50</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/fire.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="iceLabel">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/ice.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="darkLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/dark.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="dragonLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>typeimg/dragon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="normalWeak">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rockWeak">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="waterWeak">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="fightWeak">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="bugWeak">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="grassWeak">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="flyingWeak">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="ghostWeak">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="electricWeak">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="poisonWeak">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="steelWeak">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="psychicWeak">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="groundWeak">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="fireWeak">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="iceWeak">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>80</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="dragonWeak">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="darkWeak">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="fairyWeak">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>23</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>   ID                 POKEMON      </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QComboBox" name="pokeBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="idBlock">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TCG">
     <attribute name="title">
      <string>TCG</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPokeORAS">
    <property name="title">
     <string>ORAS</string>
    </property>
    <widget class="QMenu" name="menuPokeXY">
     <property name="title">
      <string>XY</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuPokeXY"/>
    <addaction name="actionHGSS"/>
   </widget>
   <widget class="QMenu" name="menuPtcgo">
    <property name="title">
     <string>PTCGO</string>
    </property>
   </widget>
   <addaction name="menuPokeORAS"/>
   <addaction name="menuPtcgo"/>
  </widget>
  <action name="actionHGSS">
   <property name="text">
    <string>HGSS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pokeBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>typeText</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idBlock</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>typeText</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
